pkg/oidc/login_handler.go

```diff
@@ -0,0 +1,166 @@
+package oidc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+	"html/template"
+	"net/http"
+
+	"github.com/skpr/cognito-auth/pkg/awscreds"
+	"github.com/skpr/cognito-auth/pkg/config"
+	"github.com/skpr/cognito-auth/pkg/oauth"
+	"github.com/skpr/cognito-auth/pkg/rand"
+)
+
+const (
+	scopes      = "openid email profile"
+	redirectURL = "http://localhost:8080"
+	successTpl  = `<!DOCTYPE html>
```

Comment:
This might be nice to move this into a `.tpl` file. https://golang.org/src/html/template/template.go?s=12335:12390#L376

Reply:
Unfortunately, an external file won't be compiled into the binary unless we use something like bindata.